### ISSUE TITLE
Question: way is this roundabout not a candidate for splitting?

### ISSUE THREAD

**Issue by osm-ToniE** [Zaczero/osm-relatify#19]
https://relatify.monicz.dev/?relation=9949516&load=1#map=19/47.996777/11.345838

It is the roundabout in the south-east corner of the route https://www.openstreetmap.org/way/838047752

The bus approaches the roundabout twice from the north leaving it to the west

**Comment by Zaczero** [comment]
It is a candidate for splitting. In fact, any intersection between valid roads is always split. The issue here likely lies with the routing engine. If you can confirm that [this](https://www.openstreetmap.org/relation/9949516) is the up-to-date route and that it is sorted correctly, I will investigate whether I can correct the engine in this case or determine if this relation would be unsuitable for Relatify.

**Comment by osm-ToniE** [comment]
Yes, it is correct and correctly sorted as a round-trip, I checked that in JOSM.
The corresponding GTFS would be: https://ptna.openstreetmap.de/gtfs/DE/single-trip.php?feed=DE-BY-MVV&release_date=&trip_id=2.T0.19-901-s23-22.2.H but this is w/o shape data

This is a newer GTFS which we did not yet sync to the route relation.

**Comment by Dimitar5555** [comment]
@osm-ToniE The roundabout can be split in Relatify. Can you still reproduce this issue?
